docker-compose.yml

```diff
@@ -50,6 +50,8 @@ services:
     expose: ["5000"]
     networks:
      - internal
+    entrypoint: ""
+    command: "./runserver.sh"
```

Comment:
Hmm, but you can't run commands that way after overriding... Whatever the situation where you found this helpful? (I'm genuinely curious--not trying to suggest this is a bad idea.)